gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=com.heapwhisperer.metrics.jvm
-version=1.0.2
+version=1.0.2-rocana2.0.0-SNAPSHOT
```

Comment:
We haven't been releasing Java libraries with every release. I could mark it as `rocana2` to indicate that it's tied to the 2.x.x line of Rocana?